### ISSUE TITLE
'Fully' support lifecycle prefixes

### ISSUE THREAD

**Issue by habi** [Zverik/every_door#486]
Recently I updated [a POI](https://www.openstreetmap.org/node/2892516176/history) which was long-time vacant and is now `planned` or better in `construction` as a new restaurant.

There have been several [issues (fixed) regarding `disused` amenities](https://github.com/Zverik/every_door/issues?q=is%3Aissue+disused), it would be cool if the 'future' states of the lifecycle prefix could also be supported by Every Door.

Maybe something like 'Mark as planned' besides the 'Mark as defunct' button?

**Comment by Zverik** [comment]
I doubt there will be a "planned" button (or else we'll end up with a dozen buttons for every prefix), but all prefixes will be supported in 6.0 (which you can download an alpha build of).

**Comment by habi** [comment]
> I doubt there will be a "planned" button

I don't think that a button is necessary, and seem to remember that in this case I added the `planned:` prefix manually in the tag-table-viewer-thingamajig.

> [...] all prefixes will be supported in 6.0 (which you can download an alpha build of).

Awesome!
Do you have a TestFlight-link available for iOS users?

**Comment by Zverik** [comment]
Oh, alas there are no iOS alpha builds, since I don't have a macbook at hand, and I use a rented server to do builds, which are quite slow. In this case you would need to wait for a beta I guess. Hopefully this month.

**Comment by Zverik** [comment]
Closing this ticket, ios testflight build will be available next week hopefully.
